manim/scene/scene.py

```diff
@@ -652,91 +653,105 @@ def get_moving_mobjects(self, *animations):
                 return mobjects[i:]
         return []
 
-    def get_time_progression(self, run_time, n_iterations=None, override_skip_animations=False):
+    def get_run_time(self, animations):
         """
-        You will hardly use this when making your own animations.
-        This method is for Manim's internal use.
-
-        Returns a CommandLine ProgressBar whose fill_time
-        is dependent on the run_time of an animation,
-        the iterations to perform in that animation
-        and a bool saying whether or not to consider
-        the skipped animations.
+        Gets the total run time for a list of animations.
 
         Parameters
         ----------
-        run_time: float
-            The run_time of the animation.
+        animations: list of Animation
+            A list of the animations whose total
+            run_time is to be calculated.
+
+        Returns
+        ------
+        float
+            The total run_time of all of the animations in the list.
+        """
 
-        n_iterations: int, optional
-            The number of iterations in the animation.
+        return np.max([animation.run_time for animation in animations])
 
-        override_skip_animations: bool, optional
-            Whether or not to show skipped animations in the progress bar.
+    def formatted_progress(self):
+        """
+        Returns a GlobalProgress formatted to display the number of elapsed
+        frames.
 
         Returns
         ------
-        ProgressDisplay
-            The CommandLine Progress Bar.
+        GlobalProgress
```

Comment:
For correct sphinx rendering
```suggestion
        :class:`GlobalProgress`
```